tests/back/api/test_runs_api.py

```diff
@@ -49,7 +53,7 @@ def test_create_run(client: TestClient, experiment_id: int):
             "description": "This is a test run",
         },
     )
-    assert response.status_code == 201, response.text
+    assert response.status_code == 201
```

Comment:
If you are looking for more detailed information for debugging when a test fails, using assert response.status_code == 200, response.text is preferable. It gives immediate insights into why the response did not meet the expected status code, which can save time during the development and debugging process

Reply:
Setting the error text returned by the API does not fully express the exception raised and may hide some useful message for diagnosing bugs. Therefore, I think it is best to disable them in all its occurences.